### PR TITLE
Migrating frameworks to use sdk 0.40.0

### PR DESCRIPTION
**Pull request by piyush-nimbalkar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+[![Build Status](https://travis-ci.org/portworx/dcos-commons.svg?branch=px_1.1_0.30.1)](https://travis-ci.org/portworx/dcos-commons)
```
Comment (disrani-px) [inline README.md]: Update branch for travis badge

File: frameworks/cassandra/universe/package.json [hunk 2]
```diff
@@ -3,14 +3,14 @@
   "upgradesFrom": ["{{upgrades-from}}"],
   "downgradesTo": ["{{downgrades-to}}"],
```
Comment (disrani-px) [inline frameworks/cassandra/universe/package.json:4]: Need to add upgrade/downgrade templates for the frameworks we've added and test that it works

**Review by disrani-px — APPROVED** [review]

File: frameworks/cassandra/universe/package.json (hunk 2 above)
Comment (piyush-nimbalkar) [reply frameworks/cassandra/universe/package.json:4]: All the frameworks are already using 4.0 packaging version, except for portworx which is using 3.0. I am not sure how we can upgrade from 3.0 to 4.0. The docs say it is not supported - https://github.com/mesosphere/universe#creating-a-package

Comment (disrani-px) [reply frameworks/cassandra/universe/package.json:4]: That picks up the service version, not the packaging version. So it's to allow upgrades from the packages that we have already released to the universe.

**Pull request merged**